awscli/topics/config-vars.rst

```diff
@@ -437,3 +437,31 @@ locale will depend on your operating system and your specific IT requirements.
 One option for UNIX systems is the ``LC_ALL`` environment variable. Setting
 ``LC_ALL=en_US.UTF-8``, for instance, would give you a United States English
 locale which is compatible with unicode.
+
+
+Plugins
+=======
+
+.. warning::
+   Plugin support in the AWS CLI v2 is completely provisional and intended to
+   help users migrate from AWS CLI v1 until a stable plugin interface is
+   released. There are no guarantees that a particular plugin or even the
+   CLI plugin interface will be supported in future versions of the AWS CLI.
```

Comment:
Let's make this reference to `AWS CLI` explicitly `AWS CLI v2` to avoid any confusion.